examples/basic-otlp/src/main.rs

```diff
@@ -92,5 +97,8 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
         });
     });
 
+    // wait for 1 seconds so that we could see metrics being pushed via OTLP every 10 seconds.
+    tokio::time::delay_for(Duration::from_secs(60)).await;
```

Comment:
Are these not pushed on drop?

Reply:
Currently, the metric export in `Exporter` trait is a sync function, so I have to use another tokio task to actually send the request. We could add a reply channel in the request and block on the reply channel to do force push/push or drop.
But no we don't have it now. Though this should come after we add timeout logic.